examples/complete/outputs.tf

```diff
@@ -1,19 +1,38 @@
-output "parent_zone_id" {
-  value = module.domain.parent_zone_id
+output "public_parent_zone_id" {
+  value = module.public_domain.parent_zone_id
 }
 
-output "parent_zone_name" {
-  value = module.domain.parent_zone_name
+output "public_parent_zone_name" {
+  value = module.public_domain.parent_zone_name
 }
 
-output "zone_id" {
-  value = module.domain.zone_id
+output "public_zone_id" {
+  value = module.public_domain.zone_id
 }
 
-output "zone_name" {
-  value = module.domain.zone_name
+output "public_zone_name" {
+  value = module.public_domain.zone_name
 }
 
-output "fqdn" {
-  value = module.domain.fqdn
+output "public_fqdn" {
+  value = module.public_domain.fqdn
+}
```

Comment:
We want to maintain backward compatibility. Let's leave these as they were previously and only add the new outputs.

Reply:
EDIT: Brain fart on my part, for some reason I thought these were the root `variables.tf`